Assets/Scripts/EOSAchievementManager.cs

```diff
@@ -489,21 +438,31 @@ public static uint GetAchievementsCount()
         /// <param name="achievementId">
         /// The id of the achievement to unlock for the current player.
         /// </param>
-        /// <param name="callback">
-        /// Invoked upon the completion of the attempt to unlock the achievement
-        /// for the given player.
-        /// </param>
-        public void UnlockAchievement(string achievementId, OnUnlockAchievementsCompleteCallback callback)
+        public Task UnlockAchievementAsync(string achievementId)
```

Comment:
Similar comment wrt `async Task`.

Reply:
Same as above - does not need to be async, as it makes no await calls within it. This does not prevent it from being called with the `await` keyword.

Reply:
Yup, you've got it right, this doesn't need to be async.